packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -34,20 +34,25 @@ class UrlLauncherPlugin extends UrlLauncherPlatform {
     UrlLauncherPlatform.instance = UrlLauncherPlugin();
   }
 
-  String _getUrlScheme(String url) => Uri.tryParse(url)?.scheme;
+  String _getUrlScheme(String url) =>
+      Uri
+          .tryParse(url)
+          ?.scheme;
```

Comment:
Can you move this all back to a single line? I don't understand why this got reformatted like that